src/utils/changeModeDetail.js

```diff
@@ -0,0 +1,25 @@
+import HallOfFame from '../components/hallOfFame/hallOfFame';
+import Rules from '../components/rules/rules';
+
+const changeModeDetail = (isHallOfFame) => {
+  const leftButton = document.querySelector('section.container button');
+  const mainContainer = document.querySelector('.main__modes-container');
+
+  if (isHallOfFame) {
+    leftButton.innerText = 'Hall of Fame';
+    leftButton.removeEventListener('click', changeModeDetail, true);
```

Comment:
This does not seem to work in Chrome or at least in the version 96. Something link this could be a solution: `leftButton.addEventListener('click', () => changeModeDetail(!isHallOfFame), { once: true });`.